transport/server.cc

```diff
@@ -212,6 +218,101 @@ cql_server::cql_server(distributed<cql3::query_processor>& qp, auth::service& au
                             seastar::format("Holds the amount of available memory for admitting new requests (max is {}B)."
                                             "Zero value indicates that our bottleneck is memory and more specifically - the memory quota allocated for the \"CQL transport\" component.", _max_request_size))),
 
+        sm::make_derive("cql_errors_total", sm::description("Counts the total number of returned CQL errors."),
+                        error_label_instances.contains(exceptions::exception_code::SERVER_ERROR)
+                        ? std::vector<sm::label_instance>({error_label_instances.at(exceptions::exception_code::SERVER_ERROR)}) : std::vector<sm::label_instance>{},
+                        [this] { return _stats.errors.contains(exceptions::exception_code::SERVER_ERROR) ? _stats.errors.at(exceptions::exception_code::SERVER_ERROR) : 0; }),
```

Comment:
I don't think you need the contains in the map, it will be initialized to zero

Reply:
But the initialization can throw.

Reply:
This is a double lookup, better to use find() and examine the returned iterator.

Reply:
I'd say it's a negligible difference when it comes to hashing ints. And this is somewhat more readable. But I can live with changing that.
Although, do we actually need all that stuff if the entries were initialized beforehand? It's true for the `error_label_instances` and can be true for the `_stats.errors` (I just set all of them to 0 in the same place I'm creating the label instances hashmap)

Reply:
@avikivity I probably missing something, if you call `_stats.errors[key]`, wouldn't it return zero if `key` is not in errors?

Reply:
It can resize the table underneath and throw if it can't do it

Reply:
At least that's the reason I'm aware of

Reply:
Yes